sites/fast-website/src/public/index.html

```diff
@@ -3,12 +3,114 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
     <fast-design-system-provider use-defaults>
-      <fast-button>Hello, World!</fast-button>
-      <gradient-picker>stub</gradient-picker>
+      <div class="wrapper">
+        <!--header-->
+        <header class="header">
+          <div>
+              <button>Logo</button>
+          </div>
+          <div class="header-link-group">
+            <button>Component</button>
+            <button>Integration</button>
+            <button>Documentation</button>
+            <button>Community</button>
```

Comment:
Assuming these will be our `fast-components` and this will change...either way, I'd expect anchors here as they navigate to different pages.

Reply:
Correct, this will be changed. When I get to actually working in this section, I will add the correct content, components, and style.

Reply:
party